### PR TITLE
Removes concept of email from kademlia metadata

### PR DESCRIPTION
**Pull request by dylanlott**

### CODE REVIEW

File: pkg/kademlia/config.go [hunk 1]
```diff
@@ -54,30 +54,17 @@ func (c Config) Verify(log *zap.Logger) error {
 
 // OperatorConfig defines properties related to storage node operator metadata
 type OperatorConfig struct {
```
Comment (egonelbre) [inline pkg/kademlia/config.go]: The email must still be specified for answering info query for trusted satellite. Also removing this is the cause for build failures.

Comment (dylanlott) [reply pkg/kademlia/config.go]: Then we can't remove emails from Kademlia until that is fixed.

File: internal/testplanet/planet.go [hunk 2]
```diff
@@ -395,7 +395,6 @@ func (planet *Planet) newSatellites(count int) ([]*satellite.Peer, error) {
 				Alpha:  5,
 				DBPath: storageDir, // TODO: replace with master db
 				Operator: kademlia.OperatorConfig{
-					Email:  prefix + "@example.com",
```
Comment (egonelbre) [inline internal/testplanet/planet.go]: operator configs should stay as it was before

File: pkg/kademlia/config.go [hunk 3]
```diff
@@ -54,8 +54,8 @@ func (c Config) Verify(log *zap.Logger) error {
...
-	Email  string `user:"true" help:"operator email address" default:""`
```
Comment (egonelbre) [inline pkg/kademlia/config.go]: keep the order as before

File: pkg/kademlia/config.go [hunk 4]
```diff
@@ -69,15 +69,6 @@ func (c OperatorConfig) Verify(log *zap.Logger) error {
 	return nil
 }
 
-func isOperatorEmailValid(log *zap.Logger, email string) error {
```
Comment (egonelbre) [inline pkg/kademlia/config.go]: keep the order as it was before

**Pull request merged**